.claude/commands/multi-agent/qk-worktree.md

```
@@ -0,0 +1,171 @@
+# Reject anything that is not a plain issue number
+if ! [[ "$ARGUMENT" =~ ^[0-9]+$ ]]; then
+  echo "❌ Invalid issue number: '$ARGUMENT'. Provide a numeric GitHub issue number, e.g. /qk-worktree 312"
+  exit 1
+fi
+```
+
+## Step 1: Verify GitHub Issue [AGENT]
+
+Verify the GitHub issue exists and is ready for work.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)